cluster/gce/util.sh

```diff
@@ -1543,6 +1562,8 @@ function build-windows-kube-env {
 WINDOWS_NODE_INSTANCE_PREFIX: $(yaml-quote ${WINDOWS_NODE_INSTANCE_PREFIX})
 NODE_BINARY_TAR_URL: $(yaml-quote ${NODE_BINARY_TAR_URL})
 NODE_BINARY_TAR_HASH: $(yaml-quote ${NODE_BINARY_TAR_HASH})
+CSI_PROXY_TAR_URL: $(yaml-quote ${CSI_PROXY_TAR_URL})
+CSI_PROXY_TAR_HASH: $(yaml-quote ${CSI_PROXY_TAR_HASH})
```

Comment:
CSI_PROXY_TAR_HASH is going to be empty if CSI_PROXY_TAR is not used to pass a local file and the CSI_PROXY_TAR_URL from config-common.sh is used. Is that ok?